dmutils/email/dm_notify.py

```diff
@@ -185,7 +183,8 @@ def send_email(
 
         except HTTPError as e:
             self._log_email_error_message(to_email_address, template_name_or_id, reference, e)
-            if any(msg["message"].startswith("Missing personalisation") for msg in e.message):
+            if isinstance(e.message, list) and \
+                    any(msg["message"].startswith("Missing personalisation") for msg in e.message):
```

Comment:
Is there no case when we'd want to raise an `EmailTemplateError` for a `str` error message?

Reply:
No - the message is only a string if the response body was not an expected error response from Notify.